project/Versions.scala

```diff
@@ -5,8 +5,8 @@ object Versions {
   val androidPlatformV = "android-21"
   val androidV = "21.0.3"
   val androidPluginV = "1.3.13"
-  val macroidExtras = "0.1.1"
-  val macroidV = "2.0.0-M3"
+  val macroidExtras = "0.2"
+  val macroidV = "2.0.0-20150427"
```

Comment:
`2.0.0-20150427` is the revision for `2.0.0-M4` in [Bintray](https://bintray.com/stanch/maven/macroid/view)